src/node.rs

```diff
@@ -134,6 +125,18 @@ impl<'hw: 'op, 'op: 'g, 'g> PartialEq for Node<'hw, 'op, 'g> {
 
 impl<'hw: 'op, 'op: 'g, 'g> Eq for Node<'hw, 'op, 'g> {}
 
+/// Directly obtaining a scalar value from a node.
+impl<'hw: 'op, 'op: 'g, 'g> From<Node<'hw, 'op, 'g>> for f32 {
+    fn from(node: Node<'hw, 'op, 'g>) -> Self {
+        node.graph
+            .borrow_mut()
+            .calculate(node.step_id)
+            .unwrap()
+            .get_scalar_f32()
+            .unwrap()
```

Comment:
we can use `Node::calculate()`:

```suggestion
        node.calculate().unwrap().get_scalar_f32().unwrap()
```